VikingQuiz.UI/src/Components/YesNoComponent/YesNoComponent.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import './YesNoComponent.css';
+
+
+class YesNoComponent extends React.Component<any, any>{
```

Comment:
A nice improvement would be to specify what the props should be.
Create an interface and declare that you'll be receiving a 'confirmationMessage' and two functions (handles for yes and no)